.github/workflows/ci.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: wyvox/action-setup-pnpm@v3
       - run: pnpm build
+      - run: pnpm run lint
```

Comment:
can you put this in its own job above test-default?

and then add the lint command to package.json? :sweat_smile: 
https://github.com/NullVoxPopuli/ember-composable-helpers/pull/13/files#diff-b03f4acb30e04ceb12a40763ef649759b3f632df2357bcec14fab6dd29ebbfb4L5